### DISCUSSION TITLE
Support for Laravel 9

### DISCUSSION THREAD

**Discussion by corpta** [rappasoft/laravel-livewire-tables#631] [Feature Requests]
I get the following error when trying to add to Laravel 9
```
rappasoft/laravel-livewire-tables v1.22.0 requires illuminate/contracts ^8.0 -> found illuminate/contracts[v8
.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
```

Assuming you can add do this to fix?

        "illuminate/contracts": "^8.0|^9.0"

**Comment by rappasoft** [comment]
Laravel 9 has not been launched yet and I haven't had time to test.

**Reply by barobba** [reply]
Has a version for Laravel 9 been released yet?

**Comment by jadersbr** [comment]
Currently I'm using this package with Laravel 9.23.
